### DISCUSSION TITLE
[Feature] Photo page does not include all the options

### DISCUSSION THREAD

**Discussion by tal-sitton** [immich-app/immich#10043] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

opening a photo does not have the same options as holding the photo from an album or from home page (like removing from album, move to trash etc.) 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Can you elaborate on what you mean? Perhaps with some screenshots?

**Reply by tal-sitton** [reply]
Sure,
When selecting an image in an album, there are options to remove from album, move to trash and more.
![Screenshot_20240608_135955.jpg](https://github.com/immich-app/immich/assets/53299816/21c7a597-0968-454c-928e-c850233066de)

But when opening the photo, I have no such options
![Screenshot_20240608_140007.jpg](https://github.com/immich-app/immich/assets/53299816/f9df7cce-e6f6-45ae-93b3-cdc8a11aef48)

**Reply by bo0tzz** [reply]
In the full size viewer, the move to trash button is in the bottom right. There is no remove from album button there, because when you're in the full size viewer there is no album context to remove it from.

**Reply by tal-sitton** [reply]
The remove button deletes the photo from the server AND from my device.
And in the full size viewer I can add to album, an option to remove from album is necessary in my opinion.

**Reply by tal-sitton** [reply]
For example,
In Google photos, when opening a photo from an album, there is an option to remove the photo from the album (it's also the first option)
![Screenshot_20240608_141932_Photos.jpg](https://github.com/immich-app/immich/assets/53299816/b80ba675-f96d-411c-8be6-9acb457ece3e)
